src/docs/get-started/install/macos.md

```diff
@@ -14,7 +14,7 @@ To install and run Flutter,
 your development environment must meet these minimum requirements:
 
 - **Operating Systems**: macOS (64-bit)
-- **Disk Space**: 700 MB (does not include disk space for IDE/tools).
+- **Disk Space**: 2.200 MB (does not include disk space for IDE/tools).
```

Comment:
Is this 2.2 GB?  I calculate 1.98 GB on my local machine.  And the previous version I calculate at 1.51 GB, so I'm curious what the 700 MB is measuring?

Reply:
I don't know, @tvolkert. I'm happy to modify it to 2 GB or even 1..98 GB. I'm not sure who performed the calculations, or how it was calculated.

Reply:
Where'd you see/get the `2.200 MB` number?

Reply:
From the bug report:  https://github.com/flutter/website/issues/2995

Reply:
I think that info is old (pre-1.9.1).  I'd update it to say `2.8 GB`

Reply:
Thanks! Will do.